packages/keystone/rateLimiting/plugin.js

```diff
@@ -0,0 +1,196 @@
+const ms = require('ms')
```

Comment:
Does you really need package that converts to milliseconds? There are only two usage of it...

Reply:
In a perfect world (out of MVP) I probably want to override values on `per-query/mutation` level...
So if I do something like...
```
fields: ...
hooks: ...
rateLimit: {
    create: {
        window: "5min"
        quota: 25
    }
}
```

Plugin will generate separate reeds key for this mutation, that's the reason I've introduced this npm package. Quality of life